Add Issues and PR templates

esau-morais opened the pull request:
## Description 

What types of changes does your code introduce to links.dev?
> :information_source: _Put a `x` in one or more boxes that might apply to it_

- [ ] Sign up (register to `links.dev`)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation (if none of the other choices apply)

## Checklist

> :information_source: _This is simply a reminder of what we are going to look for before merging your code._
> 
>  _Mark the boxes with a `x` that are related to the PR._
> If you're unsure about any of them, don't hesitate to ask. 

- [ ] I have read the ["How to sign up" section](https://github.com/fatih-yavuz/links.dev#how-to-sign-up)
- [x] CI still works with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

fatih-yavuz commented:
This looks, sweet Esau 

Can we introduce different templates per branch type? For example, the fix or feature branches can be named like `issue-${number}` so that we will ask some of the questions only on those branches:
- CI still works with my changes 
- I have added tests that prove my fix is effective or that my feature works
-  I have added the necessary documentation (if appropriate)
etc. 

For registration PRs, I would only ask the following:
- I have created my-links repository and added my page.json
- my page.json contains my links, not others
- I've updated registry.yaml

esau-morais commented:
> This looks, sweet Esau
> 
> Can we introduce different templates per branch type? For example, the fix or feature branches can be named like `issue-${number}` so that we will ask some of the questions only on those branches:
> 
> * CI still works with my changes
> * I have added tests that prove my fix is effective or that my feature works
> * I have added the necessary documentation (if appropriate)
>   etc.
> 
> For registration PRs, I would only ask the following:
> 
> * I have created my-links repository and added my page.json
> * my page.json contains my links, not others
> * I've updated registry.yaml



> This looks, sweet Esau
> 
> Can we introduce different templates per branch type? For example, the fix or feature branches can be named like `issue-${number}` so that we will ask some of the questions only on those branches:
> 
> * CI still works with my changes
> * I have added tests that prove my fix is effective or that my feature works
> * I have added the necessary documentation (if appropriate)
>   etc.
> 
> For registration PRs, I would only ask the following:
> 
> * I have created my-links repository and added my page.json
> * my page.json contains my links, not others
> * I've updated registry.yaml

I love the idea, I was actually unaware that it was possible to also create separated files for PR templates LOL, once I've never seen any org using it. Thank you

fatih-yavuz commented on .github/PULL_REQUEST_TEMPLATE/issues.md:
```diff
@@ -0,0 +1,26 @@
+>  _Mark the boxes with a `x` that are related to the PR._
+> If you're unsure about any of them, don't hesitate to ask. 
+
+- [ ] CI still works with my changes
```
The CI status already indicates that. I think it is unnecessary. What do you think? Other than that, it looks lovely. Thanks for enhancing the README as well 🚀

esau-morais replied:
Yes, I agree

esau-morais reviewed (commented):
I'd love to also have feedback from @bsoyka and @haliliceylan to proceed with this PR 😄

haliliceylan commented on README.md:
```diff
@@ -25,34 +27,39 @@ To register for links.dev, you need to do two things:
 - Create **a new public repository** called "my-links". (You can also fork [this](https://github.com/fatih-yavuz/my-links)) This repository should contain two files:
-  1. page.json, which contains the content of your page.
+  1. `page.json`, which contains the content of your page.
   2. [Optional] custom.css, which allows you to customize the look of your page.
```
custom.css => `custom.css`

esau-morais replied:
Applying changes in the next commit.

haliliceylan reviewed (changes requested):
LGTM
